new-log-viewer/src/typings/worker.ts

```diff
@@ -7,22 +7,35 @@ import {LOG_LEVEL} from "./logs";
  */
 type FileSrcType = string | File;
 
+
+/**
+ * For a page requested by the page number `CURSOR_CODE`, this enum indicates which log event
+ * number (e.g., first on page, or last on page) should be returned with the page.
+ */
+enum EVENT_POSITION {
```

Comment:
How about `EVENT_POSITION_ON_PAGE`? Alone, this enum doesn't fully make sense unless it's in the context of a page.